renovate.json

```diff
@@ -21,7 +21,7 @@
     },
     {
       "matchManagers": ["npm"],
-      "matchPackageNames": ["@types/node", "govuk-frontend"],
+      "matchPackageNames": ["@types/node", "govuk-frontend/dist"],
```

Comment:
I don't think Renovate needs to change, the package name is still govuk-frontend.